src/modules/person/persistence/person.repository.ts

```diff
@@ -170,4 +234,25 @@ export class PersonRepository {
 
         return person;
     }
+
+    public async deletePersonAndKontexte(
```

Comment:
Why don't you use deletePersonIfAllowed directly? Especially in this case it is necessary, because you can read the person with "Person verwalten" right, but you can delete it only with "Person sofort löschen".

Reply:
Done